sdk/storage/Azure.Storage.Common/src/Shared/StorageWriteStream.cs

```diff
@@ -346,6 +346,30 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
+#if NETCOREAPP3_0_OR_GREATER
```

Comment:
It seems this is also available in netstandard2.1 (which we do have explicit targets for in some packages). I think ` || NETCORESTANDARD2_1_OR_GREATER` is what you want to add.